src/content/api/cli.md

```diff
@@ -366,3 +367,15 @@ webpack --progress --profile
 6ms emitting
 ⋮
 ```
+
+## Pass CLI arguments to Node.js
+
+To pass arguments directly to Node.js process, you can use the `--node-args` option. All other flags and options will be received by the webpack-cli.
+
+For example, to increase the memory limit of Node.js process to 4 GB
+
+```bash
+webpack --node-args="--max-old-space-size=4096"
+```
+
+You can pass multiple options to Node.js: `webpack --node-args="--max-old-space-size=4096" --node-args="-r /path/to/preload/file.js"`
```

Comment:
Can you please format this example as ```bash too? Also remove `:` to be consistent with the first example. Also phrasing could be `Syntax of passing multiple options to Node.js`. I think sounds more natural for documentation rather than telling user what he can do... WDYT?

Reply:
Fixed the example
> WDYT

In the docs, there are many mentions of a user, (e.g. `This option allow you to ...`) and I think that it's ok even for docs ☺️

Reply:
Yeah they were created without certain guidelines, i just think that moving forward, we can improve the tone of voice and how we add infotmation, indeed it doesnt seem to be worth it to refactor other occurrences, but while we are adding new information, it can be improved